registry/coder/modules/agentapi/main.tf

```
@@ -126,6 +126,27 @@ variable "agentapi_port" {
   default     = 3284
 }
 
+locals {
+  # agentapi_subdomain_false_min_version_expr matches a semantic version >= v0.3.1.
+  agentapi_subdomain_false_min_version_expr = "^v(0\\.(3\\.[1-9]+|[4-9]+\\.\\d+)|[1-9]\\d*\\.\\d+\\.\\d+)$"
+}
+
+variable "agentapi_subdomain" {
```

```suggestion
variable "subdomain" {
```

I know we are prefixing `agentapi_` elsewhere, but I think it's not needed.